examples/layers/ios/.gitignore

```diff
@@ -0,0 +1,37 @@
+.idea/
```

Comment:
SGTM. @jakobr-google for the pod side of things; I'm not 100% sure what should be checked in and what is auto-generated and should be .gitignored.